cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/PushCommand.java

```diff
@@ -98,6 +98,9 @@ public class PushCommand implements BLauncherCmd {
     @CommandLine.Option(names = {"--skip-source-check"}, description = "skip checking if source has changed")
     private boolean skipSourceCheck;
 
+    @CommandLine.Option(names = "--experimental", description = "enable experimental language features")
```

Comment:
Do we need it for push command? We don't do any compilations here. We only validate the bala and push it to the central.

Reply:
removed via [8991d7c](https://github.com/ballerina-platform/ballerina-lang/pull/43950/commits/8991d7c28a89802bd8da81edd304e601c11ea440)